crates/router/src/core/user.rs

```diff
@@ -395,14 +395,20 @@ pub async fn invite_user(
 
         Ok(ApplicationResponse::Json(user_api::InviteUserResponse {
             is_email_sent: false,
+            password: None,
         }))
     } else if invitee_user
         .as_ref()
         .map_err(|e| e.current_context().is_db_not_found())
         .err()
         .unwrap_or(false)
     {
-        let new_user = domain::NewUser::try_from((request.clone(), user_from_token.clone()))?;
+        let new_user_password: Secret<String> = uuid::Uuid::new_v4().to_string().into();
+        let new_user = domain::NewUser::try_from((
+            request.clone(),
+            user_from_token.clone(),
+            new_user_password.clone(),
+        ))?;
```

Comment:
Let the password creation be handled by the `TryFrom` implementation of `NewUser`. As you anyways have created the `get_password` method on `NewUser`, you can get the password from that method.

This will eliminate the use of `CustomPassword` type and the extra parameter in the tuple.